palantir/datasets/rpc/catalog.py

```diff
@@ -570,7 +570,7 @@ def _fields(cls) -> Dict[str, ConjureFieldDefinition]:
                 "permissionPath", OptionalTypeWrapper[str]
             ),
             "record": ConjureFieldDefinition(
-                "record", OptionalTypeWrapper[DictType(str, Any)]  # type: ignore
+                "record", OptionalTypeWrapper[dict]  # type: ignore
```

Comment:
`Dict[str, Any]` instead, you can't have a value instance as a type

Reply:
Generally though you have this problem in multiple places, new conjure python doesn't use dicttype (or any of the old type markers) because they're incompatible with generics. It will be likely easier to regenerate the file you got with conjure-python 4 and use that instead of fixing manually